### PR TITLE
Update command structure

### PR DESCRIPTION
**Pull request by Norazan**

### CODE REVIEW

File: src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt [hunk 1]
```diff
@@ -1,62 +1,70 @@
...
-        "sectionon" {
-            playerAction {
-                player.canMoveSections = true
-                sender.success("Automatic TP enabled for ${player.name}")
+        ("deeperworld" / "dw") {
+            "tp"(desc = "Enables or disables automatic teleports between sections for a player") {
+                val state by booleanArg()
+                playerAction {
+                    if (state) {
+                        player.canMoveSections = true
```
Comment (0ffz) [inline src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt:25]: Use `player.canMoveSections = state`. I'd also maybe do `val msg = if(state) "enabled" else "disabled"` and have one line sending that.

File: src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt [hunk 2]
```diff
@@ -1,62 +1,70 @@
...
             }
-        }
-        "linfo" {
-            playerAction {
-                val section = WorldManager.getSectionFor(player.location)
-                if (section == null)
-                    sender.info("${player.name} is not in a managed section")
-                else
-                    sender.info("${player.name} is in section ${section.key}")
+            "linfo"(desc = "Gets information about a players section") {
```
Comment (0ffz) [inline src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt:33]: Add an alias for it, `("layerinfo" / "linfo")`.

I'm also little concerned about everyone that already knows about the linfo command. Maybe it's worth adding some basic autocomplete so people can find it more easily, might honestly be worth keeping an error message for a little bit that says `"Please use /dw linfo or /deeperworld layerinfo instead"`.

File: src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt [hunk 3]
```diff
@@ -1,62 +1,70 @@
...
+                    if (section == null)
+                        sender.info("${player.name} is not in a managed section")
+                    else
+                        sender.info("${player.name} is in section ${section.key}")
+                }
             }
-        }
-        "syncedtime" {
-            "set"{
+            "time"(desc = "<set/add>") {
```
Comment (0ffz) [inline src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt:42]: Don't worry about description here, if someone runs the command without extra args they'll see the message with subcommand info, which i think is good enough.

**Review by 0ffz — CHANGES_REQUESTED** [review]
Mostly just want to decide what to do with the existing /linfo command

File: src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt (hunk 2 above)
Comment (Norazan) [reply src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt:33]: Yeah I agree. I'm also thinking of prioritizing autocompletion for now, its kind of essential for command use in the long run.

Comment (0ffz) [reply src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt:33]: For now literally just send back a list of all the options when the command name is deeperworld

File: src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt (hunk 3 above)
Comment (Norazan) [reply src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt:42]: My idea behind this was that if you run `/dw` you see <set/add> behind "time". But idm removing it.

**Review by 0ffz — CHANGES_REQUESTED** [review]
Just one thing, could you update to the latest idofront version, I got those boolean arg updates in so we could do `/dw tp on/off`.

**Comment by Norazan** [comment]
Ready for merge now

**Review by 0ffz — APPROVED** [review]

**Pull request merged**